datafusion/proto/README.md

```diff
@@ -17,11 +17,35 @@
   under the License.
 -->
 
-# DataFusion Proto
+# Apache Arrow DataFusion Proto
```

Comment:
Not sure if it matters to also mentioned that proto backed by `prost`

Reply:
Good idea, I added a reference